cmd/sqrld/main.go

```diff
@@ -128,4 +130,20 @@ func main() {
 	var rootCmd = &cobra.Command{Use: "app"}
 	rootCmd.AddCommand(cpu, ros, net, all)
 	rootCmd.Execute()
+
+	host := "localhost"
```

Comment:
We need to wrap this up... currently it will run every time we run the sqrld command.  What I would like for you to do is potentially add a daemon flag or subcommand (like cpu, ros, etc.).  I think at first it would be best to just be a flag.  If the flag is true, run the block of code from lines 134-148, otherwise run normally.